modules/deployment-model-impl/src/main/java/org/mule/runtime/module/deployment/impl/internal/policy/DefaultPolicyInstanceProviderFactory.java

```diff
@@ -48,7 +49,7 @@ public ApplicationPolicyInstance create(Application application, PolicyTemplate
                                           PolicyParametrization parametrization) {
     return new DefaultApplicationPolicyInstance(application, policyTemplate, parametrization, serviceRepository,
                                                 classLoaderRepository, policyTemplate.getArtifactPlugins(),
-                                                extensionModelLoaderRepository);
+                                                extensionModelLoaderRepository, null);
```

Comment:
Why no context listener is passed here?

Reply:
i added it just to use in the tests. It is null-checked